test/transport.spec.ts

```diff
@@ -141,7 +206,7 @@ describe("logForwardingEnvVar", () => {
 });
 
 describe("apiKeyEnvVar", () => {
```

Comment:
Since we no longer care about whether flushMetricsToLogs is true or false here, I don't think we need both of these test cases. We just need to test that the environment variable is set.